### PR TITLE
[DESK-530] don't pass credentials in url

### PR DESCRIPTION
**Pull request by JamieRuderman**
### Description

Electron app authenticates without passing credentials in the url

### Checklist

- [x] Pull Request title is in the format `[PROJ-12345] Some useful description here...`
- [ ] ~~I have prefixed this PR with `[WIP]` if it is a Work In Progress (not ready to merge)~~
- [x] The git branch is in the format `PROJ-12345-some-useful-description-here`
- [x] I have added one or more reviewers
- [ ] ~~I have added/updated tests for any code changes~~
- [ ] ~~I have updated documentation (including README, inline comments and docs.remote.it docs)~~
- [x] This PR only contains one isolated change (bug fix, feature, etc)
- [ ] ~~Translation strings added/updated for all user-visible text~~
- [x] I have manually reviewed this PR to make sure only the files I intended to change were changed and nothing else
- [x] I have notified reviewers on Slack by @ mentioning them in the `#desktop`channel
- [x] I have moved the Jira ticket into `Resolved` state and made a note in the ticket with a link to this PR

### Test plan

1. open the remote.it app
2. go to the webpage: http://localhost:29999
3. login in the browser
4. quit the app
5. launch the app and you should be auto-signed in

### Ticket

<https://remoteit.atlassian.net/browse/DESK-530>

### CODE REVIEW

File: frontend/src/models/auth.ts [hunk 1]
```diff
@@ -9,19 +9,27 @@ import analytics from '../helpers/Analytics'
...
 const state: AuthState = {
+  loadingInitialState: true,
   authenticated: false,
   signInStarted: false,
   signInError: undefined,
   user: undefined,
 }
 
+// @ts-ignore
+window.setAuth = (credentials: UserCredentials) => {
```
Comment (danawoodman) [inline frontend/src/models/auth.ts:28]: Ouch this is unfortunate!

File: frontend/src/models/auth.ts [hunk 2]
```diff
@@ -30,14 +38,11 @@ export default createModel({
...
-
-        console.log('INIT AUTH', { backendCredentials, storedUser })
-
-        if (backendCredentials.username && backendCredentials.authHash) user = backendCredentials
+        console.log('INIT AUTH', { storedUser })
         if (storedUser) user = JSON.parse(storedUser)
       }
-      if (user) {
+      if (user?.username) {
+        console.log('WE HAVE A USER', user)
```
Comment (danawoodman) [inline frontend/src/models/auth.ts:45]: You want this kind of log in production?

Comment (JamieRuderman) [reply frontend/src/models/auth.ts:45]: Naw.... I'll take that out - thanks.

File: frontend/src/models/auth.ts (hunk 1 above)
Comment (JamieRuderman) [reply frontend/src/models/auth.ts:28]: Yes. Lesser of two evils I suppose...

**Pull request merged**